learn/contributing/contributing_docs.mdx

```diff
@@ -11,13 +11,26 @@ Our documentation aims to be:
 - **Thorough**: the documentation website should contain all information anyone needs to use Meilisearch
 - **Open source**: this is a resource by Meilisearch users, for Meilisearch users
 
+## Documentation repository and local development
+
+The Meilisearch documentation repository only stores the content of the docs website. Because the code that makes up the website lives in another repository, **it is not possible to run a local copy of the documentation**.
```

Comment:
Should we add the link to `meilisearch.com`(https://github.com/meilisearch/meilisearch.com)?

Reply:
We could, but that repo is private, so unless that changes it doesn't make much sense.